notebooks/en/function_calling_fine_tuning_llms_on_xlam.ipynb

```diff
@@ -0,0 +1,14340 @@
+{
```

Comment:
<p>Instead of adding all the imports here, we could add them when needed. This way, we can get rid of the code block. Also, it feels like it does a lot of stuff in the same block, which is difficult to understand from a reader perspective. Always consider the target audience for generating the code blocks :) (learners).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>